File: pas/6-3.pas
uses Display;//使用Display单元库
var audio:longword;
begin
audio:=LoadAudio('display.mp3');//读取音频
PlayAudio(audio);//播放音频
SetAudioVol(audio,200);//设定音量
Msgbox('正在播放音频，音量200');
end.

File: cpp/6-3.cpp
#include "disp.h"//使用Display单元库
unsigned long audio;
int main(){
audio=loadaudio("display.mp3");//读取音频
playaudio(audio);//播放音频
setaudiovol(audio,200);//设定音量
msgbox("正在播放音频，音量200");
return 0;}
